test/app

```diff
@@ -0,0 +1 @@
+Subproject commit 7fad11ddcbb7946961054772241572a2ec3028a6
```

Comment:
what is this? It has some bizarre icon in github: https://github.com/strongloop/strong-deploy-receivers/tree/initial/test

the content above isn't something I see when checking out locally

also, so the initial PR doesn't include the boilerplate, which is hard to wade through and not necessary to review, I fast forwarded the initial onto master.... but this PR doesn't seem to realize it. not sure why, I wonder if its because of the above.

Even if I try to re-PR the branch, the comparison says there is only one commit: https://github.com/strongloop/strong-deploy-receivers/compare/initial

but this PR says its too.

/cc @rmg